src/util/options.rs

```diff
@@ -259,11 +335,11 @@ options! {
     // Should we eagerly finish sweeping at the start of a collection? (not supported)
     eager_complete_sweep:  bool                 [env_var: true, command_line: true]  [always_valid] = false,
     // Should we ignore GCs requested by the user (e.g. java.lang.System.gc)?
-    ignore_system_g_c:     bool                 [env_var: true, command_line: true]  [always_valid] = false,
-    // The upper bound of nursery size.
-    max_nursery:           usize                [env_var: true, command_line: true]  [|v: &usize| *v > 0 ] = DEFAULT_MAX_NURSERY,
-    // The lower bound of nusery size.
-    min_nursery:           usize                [env_var: true, command_line: true]  [|v: &usize| *v > 0 ] = DEFAULT_MIN_NURSERY,
+    ignore_system_gc:      bool                 [env_var: true, command_line: true]  [always_valid] = false,
+    // FIXME: This is not a good way to have conflicting options -- we should refactor this
+    // The nursery size for generational plans. It can be one of Bounded or Fixed. The nursery size
+    // can be set like "Fixed:8192", for example, to have a Fixed nursery size of 8192 bytes
+    nursery:               NurserySize          [env_var: true, command_line: true]  [always_valid] = NurserySize { kind: NurseryKind::Bounded, min: DEFAULT_MIN_NURSERY, max: DEFAULT_MAX_NURSERY },
```

Comment:
What if the specified size for bounded nursery is smaller than `DEFAULT_MIN_NURSERY`? Is that valid? I feel the validation should be: `min > 0 and max > 0 and max >= min`.

Reply:
Ooh that's a good point. I'll validate the sizes in the `new()` function for `NurserySize`

Reply:
You can change the `[always_valid]` to `|v: &NurserySize| v.min > 0 && v.max > 0 && v.max >= v.min`.